### PR TITLE
Refactor basic bonfires to use assert-style tests

### PR DESCRIPTION
**Pull request by dting**

### CODE REVIEW

**Comment by dting** [comment]
#2607

**Comment by benmcmahon100** [comment]
Hey @dting thanks so much for doing this for us!I just want to let @QuincyLarson that these ones are done before merging them in an he's currently working his way through all of them

**Comment by dting** [comment]
sure no problem. I wasn't too sure about the string messages so I just did the basic ones and was waiting for feedback. Let me know what needs changes.

**Comment by benmcmahon100** [comment]
You're gonna hate hearing but I accepted a smaller fix for one of the bonfires and now this pr is in conflict :worried: I don't suppose you could stash, pull, unstash and and push for me so i can merge it. I wasn't able to get a hold of quincy just then but I'll just hope he hasn't got this far yet :p As for the tests themselves so long as it seeds and works properly on the challenge I'm happy!

**Comment by dting** [comment]
No problem. I'll do that right now.

**Comment by dting** [comment]
I rebased the changes.

**Comment by BerkeleyTrue** [comment]
I believe @QuincyLarson may be in the middle of doing this to all challenges.

**Comment by dting** [comment]
Ya he came in the chat and asked for some one to do it a week ago. I put up the PR but didn't hear any feed back so just stopped at the basic bonfires. Feel free to close the PR.

**Comment by ahstro** [comment]
Are you still checking these @QuincyLarson?

**Comment by BerkeleyTrue** [comment]
He is still working on them.

**Comment by abhisekp** [comment]
@dcsan I think you should add separate PRs for each bonfire tests because merging this one PR will invalidate many other PRs.

cc: @QuincyLarson

File: seed/challenges/basic-bonfires.json [hunk 1]
```diff
@@ -13,8 +13,8 @@
         "Make this function return true no matter what."
       ],
       "tests": [
-        "expect(meetBonfire()).to.be.a(\"boolean\");",
-        "expect(meetBonfire()).to.be.true;"
+        "assert.typeOf(meetBonfire(), 'boolean', 'meetBonfire should return a boolean');",
```
Comment (abhisekp) [inline seed/challenges/basic-bonfires.json]: You can use `assert.isBoolean`

**Comment by QuincyLarson** [comment]
@dting I'm done with the first bonfires and will submit a pull request shortly. I'd love your help with this if you're still interested. I'd love your feedback on it.

**Pull request closed without merging**